File: chroma-league/src/test/java/net/booone/chromaleague/hud/animations/EnemyBaronKillAnimationTest.java
package net.booone.chromaleague.hud.animations;

import net.booone.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyBaronKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyBaronKillAnimation() {
        new AnimationTester().testAnimation(new EnemyBaronKillAnimation());
    }
}

File: test/hud/animations/EnemyBaronKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyBaronKillAnimation} from '../../../src/hud/animations/EnemyBaronKillAnimation.js';

/**
 * Integration test for enemy baron kill animation on actual hardware.
 */
describe('EnemyBaronKillAnimation Integration', () => {
    test('playEnemyBaronKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyBaronKillAnimation());
    }, 30000);
});
